SimonImanigirimpuhwe/my-brand-website
# #5 Ft signup page #5

#### What does this PR do?

- Enable guest users to create  admin account

#### Description of Task to be completed?

-Add forms for email, password and name

- Add navigation links to the page

- Add social media links

#### How should this be manually tested?

- Navigate to the page you should be able to see the form for email, password and name

- When entered credentials in input fields you should be redirected to your account dashboard

#### Any background context you want to provide?

- To signup to the platform you must be a guest user

#### What are the relevant Trello stories?

- [#5](https://trello.com/c/KBwKOdML/5-signup-page)

#### Screenshots (if appropriate)

![signup-screenshoot](https://user-images.githubusercontent.com/54619678/93466951-57ab4080-f8ed-11ea-894d-659383a6c5b8.png)



#### Questions:

- N/A
